### PR TITLE
Changed CL to PR and adjusted related terminology

### PR DESCRIPTION
**Pull request by zibellino**
TODO: Change `pr-descriptions.md` to match current practices (eg. reference to **Jira** ticket).

### CODE REVIEW

File: review/developer/pr-descriptions.md [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
 
-Future developers will search for your CL based on its description. Someone in
+Future developers will search for your PR based on its description. Someone in
 the future might be looking for your change because of a faint memory of its
 relevance but without the specifics handy. If all the important information is
 in the code and not the description, it's going to be a lot harder for them to
-locate your CL.
+locate your PR.
 
 ## First Line {#firstline}
```
Comment (oestoer) [inline review/developer/pr-descriptions.md:16]: Should this be the title of the PR in our case instead of first line?

File: README.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ Currently this contains the following documents:
 There is some Google-internal terminology used in some of these documents, which
 we clarify here for external readers:
 
-*   **CL**: Stands for "changelist", which means one self-contained change that
+*   **PR**: Stands for "pull request", which means one self-contained change that
     has been submitted to version control or which is undergoing code review.
     Other organizations often call this a "change", "patch", or "pull-request".
```
Comment (oestoer) [inline README.md]:
```suggestion
    Other organizations often call this a "change", "patch", or "changelist".
```

File: review/developer/pr-descriptions.md [hunk 3]
```diff
@@ -124,17 +124,17 @@ The first sentence describes what's actually being done. The rest of the
...
 
-CLs can undergo significant change during review. It can be worthwhile to review
-a CL description before submitting the CL, to ensure that the description still
-reflects what the CL does.
+PRs can undergo significant change during review. It can be worthwhile to review
+a PR description before merging the PR, to ensure that the description still
+reflects what the PR does.
 
-Next: [Small CLs](small-cls.md)
+Next: [Small PRs](small-prs.md)
```
Comment (oestoer) [inline review/developer/pr-descriptions.md:140]: Should we add a section before this that clarifies what are the merging strategies we're supporting?

File: review/developer/small-prs.md [hunk 4]
```diff
@@ -1,162 +1,162 @@
...
+-   Everything the reviewer needs to understand about the PR (except future
+    development) is in the PR, the PR's description, the existing codebase, or a
+    PR they've already reviewed.
 -   The system will continue to work well for its users and for the developers
-    after the CL is checked in.
--   The CL is not so small that its implications are difficult to understand. If
-    you add a new API, you should include a usage of the API in the same CL so
+    after the PR is merged.
+-   The PR is not so small that its implications are difficult to understand. If
+    you add a new API, you should include a usage of the API in the same PR so
```
Comment (oestoer) [inline review/developer/small-prs.md:59]: For me this is an important section and addresses a few things that came up in the discussion.
`1 PR != 1 feature`
So this makes the debate about squash or not squash less relevant, and a matter of personal taste (given you don't rewrite the history of a shared branch like master)

File: review/reviewer/speed.md [hunk 5]
```diff
@@ -56,17 +56,16 @@ returning from a meeting, coming back from the breakroom, etc.
 ## Fast Responses {#responses}
 
 When we talk about the speed of code reviews, it is the _response_ time that we
-are concerned with, as opposed to how long it takes a CL to get through the
-whole review and be submitted. The whole process should also be fast, ideally,
-but **it's even more important for the _individual responses_ to come quickly
```
Comment (oestoer) [inline review/reviewer/speed.md]: This sentence got shortened a bit and for me it's not clear now with the modification. Was this intentional or accidental?

**Review by oestoer — APPROVED** [review]
Thanks Szabolcs for updating this. Looks good and I've added a few comments and questions.

File: review/developer/pr-descriptions.md (hunk 3 above)
Comment (zibellino) [reply review/developer/pr-descriptions.md:140]: @oestoer We definitely should, do you have something in mind?

File: review/reviewer/speed.md (hunk 5 above)
Comment (zibellino) [reply review/reviewer/speed.md]: fixed

File: review/developer/pr-descriptions.md (hunk 1 above)
Comment (zibellino) [reply review/developer/pr-descriptions.md:16]: It probably should, and it should probably also mention the Jira ticket references at some point too.
Shall I sketch up something?

**Review by jury89 — APPROVED** [review]
LGTM

File: review/developer/pr-descriptions.md (hunk 1 above)
Comment (oestoer) [reply review/developer/pr-descriptions.md:16]: yes, it's okay if it's just a draft, also it doesn't block this PR

File: review/developer/pr-descriptions.md (hunk 3 above)
Comment (oestoer) [reply review/developer/pr-descriptions.md:140]: yes, I'll create a separate PR for that

**Comment by zibellino** [comment]
TODO will be added as ticket to the BE Jira project (to be created).

**Pull request merged**